.github/workflows/trunk-based-flow.yml

```
@@ -0,0 +1,272 @@
+          curl -X POST "https://api.cloudflare.com/client/v4/zones/${{ secrets.CLOUDFLARE_ZONE_ID }}/purge_cache" \
+            -H "Authorization: Bearer ${{ secrets.CLOUDFLARE_API_TOKEN }}" \
+            -H "Content-Type: application/json" \
+            --data '{"purge_everything":true}' || true
+      
+      - name: Notify deployment success
+        uses: 8398a7/action-slack@v3
+        if: success()
+        with:
+          status: success
+          text: '✅ Command Center Calendar deployed to production successfully!'
+          webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)